test/parsePspTM.jl

```diff
@@ -0,0 +1,26 @@
+using Test
+using DFTK: parse_tm_file
+
+@testset "Check reading" begin
+    psp = parse_tm_file("/Users/jasonlehto/git/Dino.jl/dev/DFTK/data/psp/tm/69tm.pspnc")
```

Comment:
I guess Dino is the name of your new package then :smile:.

Reply:
Yes, I believe that eventually it will be made public. Once some more polishing happens. It's kind of a mess right now.